templates/mobile_view_groupselect.mustache

```diff
@@ -0,0 +1,69 @@
+{{=<% %>=}}
+    <ion-card>
+        <button
+            ion-button full 
+            style="margin:0" 
+            core-site-plugins-new-content 
+            component="mod_groupselect"
+            method="add_group"
+            [args]="{cmid: <%cmid%>}" 
+            title="Create New Group">
+            Create New Group
+        </button>
+    </ion-card>
+
+<ion-card *ngFor="let group of CONTENT_OTHERDATA.groups"
+    core-site-plugins-new-content >
+    <!-- Header -->
+    <ion-item style="background-color: #f7f7f7;">
```

Comment:
If possible do all styling in the stylesheets. We should only continue with this styling if we dynamically bind angular styles. Even so we can dynamically bind classes.